app/main/views/direct_award.py

```diff
@@ -12,7 +12,27 @@
 from ...models.direct_award import DirectAwardProject, DirectAwardSearch
 from ...utils import (
     get_json_from_request, get_int_or_400, json_has_required_keys, pagination_links,
-    get_valid_page_or_1, validate_and_return_updater_request, drop_all_other_fields)
+    get_valid_page_or_1, validate_and_return_updater_request, drop_all_other_fields, RANDOM_ID_BIT_LENGTH)
+
+
+def get_project_by_id(project_id: int):
+    """During the transitional period, project_id may be either the internal or the public ID of the project."""
```

Comment:
We think we'll probably maintain this functionality going forwards, so the comment needs a little adjustment.

Arguably the function name could also be more clear, that it is intended to do something with multiple kinds of id. Somehow :)